frontend/src/views/Core/ForgotPassword/ForgotPassword.scss

```diff
@@ -0,0 +1,31 @@
+@import "src/scss/Core";
+
+#forgot-password-page {
+  #navbar {
```

Comment:
As you said, please create a **sidebar** component and move the css of these components to **Core** scss
Create the sidebar inside _core/components_ for the js and for css move to _scss/Core.scss_.